### ISSUE TITLE
problems with aggregate columns in datagrid

### ISSUE THREAD

**Issue by lucamorelli** [stsrki/Blazorise#809]
I have this grid

```
<DataGrid TItem="myData" Data="@listadati" PageSize="15">
    <DataGridAggregates>
        <DataGridAggregate TItem="myData" Field="@nameof( myData.Mese)" Aggregate="DataGridAggregateType.None" />
        <DataGridAggregate TItem="myData" Field="@nameof( myData.Venduto)" Aggregate="DataGridAggregateType.Sum" DisplayFormat="{0:C}" DisplayFormatProvider="@System.Globalization.CultureInfo.GetCultureInfo("it-IT")" />
        <DataGridAggregate TItem="myData" Field="@nameof( myData.Acquistato)" Aggregate="DataGridAggregateType.Sum" DisplayFormat="{0:C}" DisplayFormatProvider="@System.Globalization.CultureInfo.GetCultureInfo("it-IT")" />
        <DataGridAggregate TItem="myData" Field="@nameof( myData.Saldo)" Aggregate="DataGridAggregateType.Sum" DisplayFormat="{0:C}" DisplayFormatProvider="@System.Globalization.CultureInfo.GetCultureInfo("it-IT")" />
        <DataGridAggregate TItem="myData" Field="@nameof( myData.Percentuale)" Aggregate="DataGridAggregateType.None" />
    </DataGridAggregates>
    <DataGridColumns>
        <DataGridCommandColumn TItem="myData" />
        <DataGridColumn TItem="myData" Field="@nameof(myData.Mese)" Caption="Mese" Sortable="false" />
        <DataGridColumn TItem="myData" Field="@nameof(myData.Venduto)" Caption="Vendite" Sortable="false" />
        <DataGridColumn TItem="myData" Field="@nameof(myData.Acquistato)" Caption="Acquisti" Sortable="false" />
        <DataGridColumn TItem="myData" Field="@nameof(myData.Saldo)" Caption="Saldo" Sortable="false" />
        <DataGridColumn TItem="myData" Field="@nameof(myData.Percentuale)" Caption="%" Sortable="false" />
    </DataGridColumns>
</DataGrid>

```
after the latest upgrade the visualization of the aggregates columns is broken and are shown like this

![WhatsApp Image 2020-04-23 at 14 52 29 (1)](https://user-images.githubusercontent.com/127777/80103533-a9d17b00-8574-11ea-85eb-850de08fc251.jpeg)

Any change to do?

**Comment by stsrki** [comment]
Based on the image I see that aggregate cells are being offset. Can you confirm that's the issue?

**Comment by lucamorelli** [comment]
I suppose same thing, but I don't understand why, because  we have the same number of rows in the datagridaggregates, and was working fine before the last update

**Comment by stsrki** [comment]
I believe this should be fixed in the last 0.9.3.x version. We have done a lot of improvements. If it is still not working, please reopen it.
